ra_aid/proc/interactive.py

```
@@ -1,90 +1,158 @@
-        raise RuntimeError("Error running interactive capture") from e
-    finally:
-        # Ensure files are removed no matter what
-        cleanup()
-
-    return output, return_code
+    try:
+        stdin_fd = sys.stdin.fileno()
+    except (AttributeError, io.UnsupportedOperation):
+        stdin_fd = None
+
+    proc = subprocess.Popen(
```

Ruff (0.8.2)

`subprocess` call: check for execution of untrusted input

(S603)